starknet_py/net/account/account.py

```diff
@@ -110,7 +111,7 @@ async def _get_max_fee(
 
         if auto_estimate:
             estimate_fee = await self._estimate_fee(transaction)
-            max_fee = int(estimate_fee.overall_fee * 1.1)
+            max_fee = int(estimate_fee.overall_fee * ESTIMATED_FEE_MULTIPLIER)
```

Comment:
I think we could make `fee_multiplier` a class variable, but don't expose it in constructor. This way `Account` would use default of `1.5` but could adjusted by the user like `account.fee_multiplier = 2.0`.

cc @war-in

Reply:
Yes, I would like to have it too